wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'wagtailadmin/shared/headers/slim_header.html' %}
+{% load wagtailadmin_tags i18n %}
+
+{% block header_content %}
+    {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' %}
+
+    <h1 class="w-sr-only">
+        {{ title }}
+    </h1>
+    {# Actions divider #}
+    <div class="w-w-px w-h-[30px] w-ml-auto sm:w-ml-0 w-bg-grey-100"></div>
+    {# Page actions dropdown #}
+    {% page_header_buttons parent_page page_perms=page_perms %}
+{% endblock %}
+{% block actions %}
+    {% with nav_icon_classes='w-w-4 w-h-4' nav_icon_button_classes='w-h-[50px] w-bg-transparent w-box-border w-py-3 w-px-3 w-flex w-justify-center w-items-center w-outline-offset-inside w-text-grey-400 w-transition hover:w-transform hover:w-scale-110 hover:w-text-primary focus:w-text-primary' %}
+        {% if title != "Root" %}
+            {% include "wagtailadmin/shared/side_panel_toggles.html" %}
+        {% endif %}
+        {# Page history #}
+        {% if parent_page.get_latest_revision %}
+            <a href="{% url 'wagtailadmin_pages:history' parent_page.id %}"
+                class="{{ nav_icon_button_classes }}"
+                data-tippy-content="{% trans 'History' %}"
+                data-tippy-offset="[0, 0]"
+                data-tippy-placement="bottom"
+                aria-label="{% trans 'History' %}"
+            >
+                {% icon name="history" class_name=nav_icon_classes %}
+            </a>
+        {% endif %}
+
+        {% include "wagtailadmin/shared/page_status_tag_new.html" with page=parent_page %}
+
```

Comment:
Looks like the most correct way to check for this is `is_root`. And I think we don’t want any of those things for the root page, so could wrap everything in the `if`:

```suggestion
        {% if not parent_page.is_root %}
            {% include "wagtailadmin/shared/side_panel_toggles.html" %}
		    {# Page history #}
		    {% if parent_page.get_latest_revision %}
		        <a href="{% url 'wagtailadmin_pages:history' parent_page.id %}"
		            class="{{ nav_icon_button_classes }}"
		            data-tippy-content="{% trans 'History' %}"
		            data-tippy-offset="[0, 0]"
		            data-tippy-placement="bottom"
		            aria-label="{% trans 'History' %}"
		        >
		            {% icon name="history" class_name=nav_icon_classes %}
		        </a>
		    {% endif %}
		    
		    {% include "wagtailadmin/shared/page_status_tag_new.html" with page=parent_page %}
        {% endif %}
```